build.gradle

```diff
@@ -21,7 +21,7 @@ targetCompatibility = 1.8
 
 group = "io.github.lxgaming"
 archivesBaseName = "Sumsang"
-version = "1.14.4-1.2.1"
+version = "forge-1.15.1-30.0.36"
```

Comment:
Please change this to `1.15.1-1.2.1`, This version field represents the mod version not the Forge version.